src/core/api/payment_manager_api/pm-per-nodo/v2/_base_policy.xml.tpl

```diff
@@ -1,42 +0,0 @@
-<policies>
-    <inbound>
-      <base />
-      <ip-filter action="forbid">
```

Comment:
It's better to have the IP filter in the base policy so it's default for all APIs, what do you think?